lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.SpaceType.rb

```diff
@@ -441,22 +451,38 @@ def space_type_light_sch_change(model)
     # set schedule for lighting
     schedule_hash = {}
     model.getSpaces.each do |space|
-      space_type = space.spaceType.get
-      if space_type.hasAdditionalProperties && space_type.additionalProperties.hasFeature('regulated_lights_name')
+      space_type = prm_get_optional_handler(space, @sizing_run_dir, 'spaceType')
+      if has_additional_feature(space_type, 'regulated_lights_name')
```

Comment:
method abstraction unnecessary

Reply:
Agreed. Remove the `has_additional_feature`.